{"collections":[{"fullName":"System.Threading.Tasks.TaskCanceledException","methods":[{"name":"(System.Threading.Tasks.TaskCanceledException) object","type":"CAST","count":22},{"name":"System.Threading.Tasks.TaskCanceledException(System.String message)","type":"CTOR","count":32},{"name":"System.Threading.Tasks.TaskCanceledException.Task","type":"PROP","count":24},{"name":"System.Threading.Tasks.TaskCanceledException(System.Threading.Tasks.Task task)","type":"CTOR","count":18},{"name":"System.Threading.Tasks.TaskCanceledException()","type":"CTOR","count":20},{"name":"System.Threading.Tasks.TaskCanceledException(System.String message, System.Exception innerException)","type":"CTOR","count":4}]}]}